### PR TITLE
Add Xforms

### PR DESCRIPTION
**Pull request by rhyspoulton**
This PR adds XForms a graphical user interface toolkit for X Window Systems.

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/Xform/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,69 @@
+{% set name = "xforms" %}
+{% set version = "0.9999.3" %}
+{% set tag = "v0r9999p3" %}
```
Comment (duncanmmacleod) [inline recipes/Xform/meta.yaml:3]: Is this used at all in the recipe? If not, it should probably be removed.

Comment (rhyspoulton) [reply recipes/Xform/meta.yaml]: Indeed `tag` is not used, so I have removed it from the meta.yaml.

**Comment by fcarbogn** [comment]
I am willing to be listed as maintainer for those recipes

**Comment by rhyspoulton** [comment]
@conda-forge/help-c-cpp , this recipe is ready for review, thanks.

**Comment by rhyspoulton** [comment]
@conda-forge-admin, please ping team

File: recipes/Xform/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    - xorg-libx11
+    - xorg-libxfixes  # [linux]
+
+test:
+  requires:
+    - pkg-config
+  commands:
+    - test -x ${PREFIX}/bin/fdesign  # [unix]
+    - test -f ${PREFIX}/lib/libfds.a  # [unix]
+    - test -f ${PREFIX}/lib/libforms.a  # [unix]
```
Comment (xhochy) [inline recipes/Xform/meta.yaml:47]: We prefer static linkage on conda-forge, can you switch this to producing `so`s?

Comment (rhyspoulton) [reply recipes/Xform/meta.yaml]: @xhochy, that makes sense. Okay, I have switched this to producing `.so`'s.

**Comment by rhyspoulton** [comment]
Ping @conda-forge/staged-recipes, is someone able to review and merge this recipe? Thanks.

**Pull request merged**